### PR TITLE
Mathjax + Cross posting

### LINKED ISSUE #62: Technical blog posts, add Nicolas Hug Gradient Boosting Descent post
Introducing a Technical category 
http://nicolas-hug.com/blog/gradient_boosting_descent

### PR DESCRIPTION
**Pull request by francoisgoupil**
- Enable MathJax support to Jekyll in order to print mathematical equations on blog posts.

- Cross post Nicolas Hug's article on "Understanding Gradient Boosting as a Gradient Descent"

Closes #62

### CODE REVIEW

**Comment by reshamas** [comment]
@francoisgoupil Thank you for this pull request. 

Any chance you could remove the French award files since that's in a separate PR? Perhaps the updated files got into two branches.

**Comment by reshamas** [comment]
Thanks @francoisgoupil

**Comment by ogrisel** [comment]
@francoisgoupil since the merge of the line-endings PR, I think it's best to start off of a new branch of an up-to-date `main` and redo your changes manually and open a new PR.

**Comment by francoisgoupil** [comment]
Ok I will dot this, thanks @ogrisel

**Comment by reshamas** [comment]
@francoisgoupil 
I removed the extra French Award blog files and tried to submit here.

So, it looks like the math formulas are not rendering here.
@laurburke @jmloyola 
Would you have any ideas on how to get the math formulas working here?
Do we need to add a Jekyll plugin in the configuration file?

**Comment by jmloyola** [comment]
To use MathJax we need to include the Javascript that does all the work ([MathJax documentation](http://docs.mathjax.org/en/latest/web/start.html#using-mathjax-from-a-content-delivery-network-cdn)).

If we want to include the script in every page of the blog (note that it may slow down pages without LaTeX, I don't know how much though), we can edit the file `blog/_includes/head/custom.html` adding:
```html
<script type="text/javascript" id="MathJax-script" async
  src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js">
</script>
```
This will load an specific version of MathJax. There are other urls we can use to load the latest version but we'll need to check the documentation.
Also, MathJax can be configured in different ways, but I think with the defaults we'll be OK.

**Comment by jmloyola** [comment]
Looking at the PR (should have done that earlier) everything looks OK here.
The script for MathJax is loaded only when needed (using `page.usemathjax`).
Maybe the math formulas were not rendering because of the URL to the MathJax script.
Or maybe the place where the URL was inserted was not correct (`_includes/masthead.html`).
I'm not sure. We'll need to debug this to find out.

I know you already solved this, but if you start seeing loading delays in pages without math it might be related to always loading the MathJax script.
In that case, we have to do something similar to what this PR did.
